Packs/CTIX/Integrations/CTIX/CTIX.py

```diff
@@ -0,0 +1,423 @@
+
+    enhanced = argToBoolean(args.get('enhanced'))
+    response = client.get_url_details(url_array, enhanced)
+    url_list = response.get("data", {}).get("results", {})
+    url_data_list, url_standard_list = [], []
+    for url_data in url_list:
+        score = to_dbot_score(url_data.get("score", 0))
+        dbot_score = Common.DBotScore(
+            indicator=url_data.get("name2"),
+            indicator_type=DBotScoreType.URL,
+            integration_name='CTIX',
+            score=score,
+        )
+        url_standard_context = Common.URL(
+            url=url_data.get("name2"),
+            dbot_score=dbot_score
+        )
+        url_standard_list.append(url_standard_context)
+        url_data_list.append(url_data)
+
+    readable_output = tableToMarkdown('URL List', url_data_list, removeNull=True)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='CTIX.URL',
+        outputs_key_field='name2',
+        outputs=url_data_list,
+        indicators=url_standard_list
+    )
+
+
+def file_details_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    file command: Returns FILE details for a list of FILE
+    """
+    file_string = args.get('file')
+    file_array = argToList(file_string)
+    invalid_hashes = []
+    for file in file_array:  # Check for Valid File Inputs
+        if not REGEX_MAP['hash'].match(file):
+            invalid_hashes.append(file)
+
+    if invalid_hashes:
+        return_warning('The following Hashes were found invalid: {}'.format(', '.join(invalid_hashes)))
+
+    enhanced = argToBoolean(args.get('enhanced'))
+    response = client.get_file_details(file_array, enhanced)
+    file_list = response.get("data", {}).get("results", {})
+    file_data_list, file_standard_list = [], []
+    for file_data in file_list:
+        score = to_dbot_score(file_data.get("score", 0))
+        dbot_score = Common.DBotScore(
+            indicator=file_data.get("name2"),
+            indicator_type=DBotScoreType.FILE,
+            integration_name='CTIX',
+            score=score
+        )
+        file_standard_context = Common.File(
+            name=file_data.get("name2"),
+            dbot_score=dbot_score
+        )
+        file_standard_list.append(file_standard_context)
+        file_data_list.append(file_data)
+
+    readable_output = tableToMarkdown('File List', file_data_list, removeNull=True)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='CTIX.File',
+        outputs_key_field='name2',
+        outputs=file_data_list,
+        indicators=file_standard_list
+    )
+
+
+def main() -> None:
+
+    base_url = demisto.params().get('base_url')
+    access_id = demisto.params().get('access_id')
+    secret_key = demisto.params().get('secret_key')
+    verify = demisto.params().get('insecure')
+    proxy = demisto.params().get('proxy', False)
+    proxies = handle_proxy() if proxy else {}
```

Comment:
```suggestion
    proxies = handle_proxy()
```
this command is required also when not running with proxy (it makes also sure that the proxy environment variables are empty in case that proxy should not be used).